src/todos.js

```diff
@@ -0,0 +1,30 @@
+export default class Todos {
+  constructor() {
+    this.list = localStorage.getItem('todos')
+      ? JSON.parse(localStorage.getItem('todos'))
+      : [];
+  }
+
+  addTodo(todo) {
+    this.list.push(todo);
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  removeTodo(todoID) {
+    this.list = this.list.filter((todo) => todo.id !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  editTodo(todoId, todoDescription) {
+    const newData = this.list.map((todo) => {
+      if (todo.id === todoId) {
+        return { ...todo, description: todoDescription };
+      }
+      return todo;
+    });
+    localStorage.setItem('todos', JSON.stringify(newData));
+  }
+}
```

Comment:
- [ ] Good work up to this point but there are some crucial aspects of the application missing
  - When editing and deleting tasks in your app, the changes are not saved in the local storage. Any changes should be saved in the local storage to ensure that consistent data is displayed when the page is refreshed. Kindly address that.

Reply:
This issue is well resolved! ✔️